packages/dynamite/dynamite/lib/src/openapi_builder.dart

```diff
@@ -1786,7 +1796,11 @@ TypeResult resolveType(
                             ],
                           ],
                           'try {',
-                          'result.${fields[result.name]!} = ${result.deserialize('data', toBuilder: true)};',
+                          if (result is TypeResultBase || result is TypeResultEnum) ...[
+                            'result._${fields[result.name]!} = ${result.deserialize('data')};',
+                          ] else ...[
+                            'result._${fields[result.name]!} = ${result.deserialize('data')}.toBuilder();',
+                          ],
```

Comment:
Is the new `_` intended? I think this breaks the code...

Reply:
But CI passes which confuses me

Reply:
I can check again but it's intended.
When we use it without the _ the setter will be called initializing the private variable with an empty builder.
This is so one could do `result.field.replace(value)` instead of `result._field = value.toBuilder()`.
The problem is that building the result will also build the instantiated builder but as it might be empty (because the catch case was triggered) it might fail with required fields not being set.

Does this make sense?

---

I'll check again tomorrow as I might have overlooked something.

Reply:
I think I understand. I just didn't know that built_value has these private setters. Thanks for the explanation!